.github/workflows/remote-release-trigger.yml

```diff
@@ -0,0 +1,45 @@
+name: Remote Release Trigger
+on:
+  repository_dispatch:
+    types: [ cli-release ]
+jobs:
+  remote-release-trigger:
+    runs-on: ubuntu-22.04
+    environment: CLI Automated Release
+    steps:
+      - name: Generate App Installation Token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92 # pin@v1
+        with:
+          app_id: ${{ secrets.CLI_RELEASE_APP_ID }}
+          private_key: ${{ secrets.CLI_RELEASE_PRIVATE_KEY }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get previous tag
+        id: previoustag
+        uses: WyriHaximus/github-action-get-previous-tag@385a2a0b6abf6c2efeb95adfac83d96d6f968e0c # pin@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get the next minor version
+        id: semvers
+        uses: "WyriHaximus/github-action-next-semvers@d079934efaf011a4cf8912d4637097fe3\
+          5d32b93" # pin@v1
```

Comment:
![image](https://user-images.githubusercontent.com/121905282/224429058-df9286c4-3175-427c-946e-3f58aa503fce.png)
This seems illegal by GitHub editor. Is `\` a standard way to do "continue on next line" in yml files?

Reply:
Fixed; looks like my editor tried to format it for me 🙂